com.unity.render-pipelines.high-definition/Editor/PostProcessing/MotionBlurEditor.cs

```diff
@@ -55,38 +53,31 @@ public override void OnInspectorGUI()
 
             PropertyField(m_MaxVelocityInPixels);
             PropertyField(m_MinVelInPixels);
+            PropertyField(m_DepthCmpScale);
 
-            if (BeginAdditionalPropertiesScope())
-            {
-                PropertyField(m_DepthCmpScale);
+            EditorGUILayout.Space();
+            EditorGUILayout.LabelField("Camera Velocity", EditorStyles.miniLabel);
```

Comment:
This gets shown when Additional Properties are toggled off and shown twice When Additional Properties are toggled on.
https://user-images.githubusercontent.com/50582134/109020061-4ba88380-76c2-11eb-9c9f-039d55c395c6.mp4

Reply:
Fixed, could you check please? Thank you :)